tools/libboard/board_n5010/board_n5010.c

```diff
@@ -231,23 +175,105 @@ fpga_result print_sec_info(fpga_token token)
 	return print_sec_common_info(token);
 }
 
+uint64_t macaddress_uint64(const struct ether_addr *eth_addr)
```

Comment:
Are there standard library API's that can be used for this (accounting for endianness)?

Reply:
I am not aware of any standard API. I took the code from board_n6000.c , which was written by @anandaravuri. We might consider moving it to board_common.c to share it between board_5010.c and board_n6000.c.

Reply:
Sounds like a good refactoring.

Reply:
I have pushed the refactoring.